dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -21,14 +22,19 @@ type InlineProps = {
 	index: number;
 	shouldHideReaderRevenue?: boolean;
 	isPaidContent?: boolean;
+	hasPageskin?: boolean;
 };
 
+// TODO move to commercial
```

Comment:
Was this TODO intended for this PR?

Reply:
I will pick this up separately as I didn't want to block this PR for it.